apollo-portal/src/main/resources/static/scripts/directive/delete-namespace-modal-directive.js

```diff
@@ -39,24 +37,18 @@ function deleteNamespaceModalDirective($window, $q, $translate, toastr, AppUtil,
                 //1. check operator has master permission
                 checkPermission(toDeleteNamespace).then(function () {
 
-                    //2. check namespace's master branch has not instances
-                    if (!checkMasterInstance(toDeleteNamespace)) {
-                        return;
-                    }
-
-                    //3. check namespace's gray branch has not instances
-                    if (!checkBranchInstance(toDeleteNamespace)) {
-                        return;
-                    }
+                    if(toDeleteNamespace.isLinkedNamespace){
+                        //2. check namespace's master branch has not instances
+                        if (!checkMasterInstance(toDeleteNamespace)) {
+                            return;
+                        }
 
-                    if (!toDeleteNamespace.isPublic || toDeleteNamespace.isLinkedNamespace) {
-                        showDeleteNamespaceConfirmDialog();
-                    } else {
-                        //5. check public namespace has not associated namespace
-                        checkPublicNamespace(toDeleteNamespace).then(function () {
```

Comment:
Shall we keep the logic to check public namespace has no associated namespace? I'm afraid there will be many orphan linked namespaces if we delete the public namespace directly.

Reply:
As above, a uniform check has been added